django/src/rdwatch_scoring/views/vector_tile.py

```diff
@@ -0,0 +1,234 @@
+            'geomfromtext', 3857, function='ST_Transform', output_field=GeometryField()
+        )
+        mvtgeom = Func(
+            'transformedgeom',
+            envelope,
+            function='ST_AsMVTGeom',
+            output_field=Field(),
+        )
+        geomfromuniongeometrytext = Func(
+            'union_geometry', 4326, function='ST_GeomFromText', output_field=Field()
+        )
+        geomfromobservationtext = Func(
+            'geometry', 4326, function='ST_GeomFromText', output_field=Field()
+        )
+
+        site_queryset = (
+            Site.objects.filter(evaluation_run_uuid=evaluation_run_uuid)
+            .alias(geomfromtext=geomfromuniongeometrytext)
+            .alias(transformedgeom=transform)
+            .filter(intersects)
+            .values()
+            .annotate(
+                id=F('site_id'),
+                mvtgeom=mvtgeom,
+                configuration_id=F('evaluation_run_uuid'),
+                label=Case(
+                    When(
+                        Q(status_annotated__isnull=False),
+                        Lower(Replace('status_annotated', Value(' '), Value('_'))),
+                    ),
+                    default=Value('unknown'),
+                ),  # This needs a version to be scoring coloring,
+                # but that needs some coordination with kitware
+                timestamp=ExtractEpoch('evaluation_run_uuid__start_datetime'),
+                timemin=ExtractEpoch('start_date'),
+                timemax=ExtractEpoch('end_date'),
+                performer_id=F('originator'),
+                region=F('region_id'),
+                groundtruth=Case(
+                    When(
+                        Q(originator='te'),
+                        True,
+                    ),
+                    default=False,
+                ),
+                site_polygon=Value(False, output_field=BooleanField()),
+            )
+        )
+        (
+            site_sql,
+            site_params,
+        ) = site_queryset.query.sql_with_params()
+
+        observations_queryset = (
+            Observation.objects.filter(
+                site_uuid__evaluation_run_uuid=evaluation_run_uuid
+            )
+            .alias(geomfromtext=geomfromobservationtext)
+            .alias(transformedgeom=transform)
+            .filter(intersects)
+            .values()
+            .annotate(
+                id=F('uuid'),
+                mvtgeom=mvtgeom,
+                configuration_id=F('site_uuid__evaluation_run_uuid'),
+                site_number=F('site_uuid__site_id'),
+                label=Case(
+                    When(
+                        Q(phase__isnull=False),
+                        Lower(Replace('phase', Value(' '), Value('_'))),
+                    ),
+                    default=Value('unknown'),
+                ),  # This should be an ID, on client side can make it understand this
+                area=Area(Transform('transformedgeom', srid=6933)),
```

Comment:
Within here we will also need:
```suggestion
                siteeval_id=F('site_uuid'),
```
This is used for clicking on a site and getting the details of it.  

We don't seem to have a site_evaluation details.  I.E `/api/scoring/observations/{uuid}`.  I don't think it's needed for this PR, but it will be required for getting images to download.

Reply:
I can implement that endpoint as part of the satellite image work.